api.php

```diff
@@ -361,7 +362,14 @@
      */
     $height = san($data['height']);
     $block = san($data['block']);
-    $ret = $trx->get_transactions($height, $block);
+    $includeMiningRewards = san($data['includeMiningRewards'] ?? '');
```

Comment:
This could be simplified to:
```php
$includeMiningRewards = (
    isset($data['includeMiningRewards']) &&
    !($data['includeMiningRewards'] === '0' || $data['includeMiningRewards'] === 'false')
);

$ret = $trx->get_transactions($height, $block, $includeMiningRewards);
```

I feel the default should be to not include them, for backwards compatibility.